portal-frontend/src/app/features/applications/edit-submission/parcel-details/parcel-entry/parcel-entry.component.html

```diff
@@ -307,9 +308,8 @@ <h6 class="owner-search-option" *ngIf="filteredOwners.length">OR</h6>
         <mat-radio-group
           color="primary"
           [value]="selectedOwner?.uuid"
-          required
+          [required]="true"
```

Comment:
This will set `required` on the `mat-radio-group` to the boolean `true` rather than a string that says "true", correct? What is the purpose of doing it this way (seems like it is not most other places)?

Reply:
Having just required without a value was showing a red error on my IDE saying that a value was required. I'm not even sure it makes a difference.